data_steward/utils/bq.py

```diff
@@ -302,6 +302,18 @@ def query(q, project_id=None, use_cache=False):
     return client.query(q, job_config=query_job_config).to_dataframe()
 
 
+def execute(client, query):
+    """
+    Execute a bigquery command and return the results in a dataframe
+
+    :param client: an instantiated bigquery client object
+    :param query: the query to execute
+    :return pandas dataframe object
+    """
+    print(query)
+    return client.query(query).to_dataframe()
+
+
```

Comment:
@nishanthpp93 I'm not sure this is the best place for this utility function.  The "utilities" in here are generally used by the pipeline (and possibly GAE).  We don't have a need to `print` queries from the pipeline.  We've also set a deprecation warning for the (very similar) `query` function in this module.  However, the function is nice to have for the analytics notebooks so we can see what is actually being sent by the notebook when running analyses.  I'd suggest creating a `data_steward/analytics/cdr_ops/utils.py` script.  It would have only a very few possible utilities: for now the `CLIENT` and `execute` definitions.

Thoughts @mark-velez and @nishanthpp93 ?

Reply:
Sounds good. I think the client is best imported from bq.py since it has support for impersonation etc. which we can leverage in the future. I have moved the execute function to the suggested utils script.